docs/modules/user-guide/partials/proc_deploying-a-devfile-registry-with-helm-chart.adoc

```diff
@@ -0,0 +1,26 @@
+[id="deploying-a-devfile-registry-with-helm-chart_{context}"]
+= Deploying a devfile registry with Helm chart
+
+[role="_abstract"]
+If the Operator Lifecycle Manager (OLM) is not installed on your Kubernetes cluster, use the devfile registry Helm chart in order to use the devfile registry operator to deploy the devfile registry.
+
+.Procedure
+. Clone the repository containing the Helm chart:
++
+[sub="+quotes"]
+----
+$ git clone https://github.com/devfile/registry-support.
+----
++
+. Navigate to the `deploy/chart/devfile-registry` directory.
+. Open the `values.yaml` file in an editor and make the following changes:
+.. Set `devfileIndex.image` to the image containing your devfile stacks.
+.. Set `devfileIndex.tag` to the image tag for your devfile index image.
+.. If installing on Kubernetes, set `global.ingress.domain` to your cluster’s ingress domain.
```

Comment:
Ingress is capitalized for Kubernetes purposes. More info here: https://kubernetes.io/docs/concepts/services-networking/ingress/